package.json

```diff
@@ -22,7 +22,7 @@
     "@wagmi/core": "^2.6.5",
     "@walletconnect/ethereum-provider": "^2.11.2",
     "@web3modal/wagmi": "^4.0.13",
-    "antd": "^5.15.0",
+    "antd": "^5.17.0",
```

Comment:
had to update so that the "wrap" in Flex is supported https://ant.design/components/flex#api